[strace] Unable to build from source

The same error occurs on armv7l and x86_64:
```
configure: error: in `/usr/local/tmp/crew/strace-5.2.tar.xz.dir/strace-5.2':
configure: error: failed to find _UPT_create in libunwind-ptrace
See `config.log' for more details
strace failed to build: `./configure --with-libunwind --prefix=/usr/local --libdir=/usr/local/lib64` exited with 1
```

--- a/packages/strace.rb
+++ b/packages/strace.rb
@@ -3,20 +3,32 @@ require 'package'
 class Strace < Package
   description 'strace is a diagnostic, debugging and instructional userspace utility for Linux.'
   homepage 'https://strace.io/'
-  version '5.2'
+  version '5.2-1'
   source_url 'https://strace.io/files/5.2/strace-5.2.tar.xz'
   source_sha256 'd513bc085609a9afd64faf2ce71deb95b96faf46cd7bc86048bc655e4e4c24d2'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/strace-5.2-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/strace-5.2-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/strace-5.2-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/strace-5.2-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '2affe33b6758ed49fc50d140e7673f44e33fc72410597eb851fef5e9699a1afc',
+     armv7l: '2affe33b6758ed49fc50d140e7673f44e33fc72410597eb851fef5e9699a1afc',
+       i686: '233884ae2118bd1558f85a0bef68260bdc981075d7e5861233004c920438f3c4',
+     x86_64: 'e5e6704b6749417a141f0c925294e53c1f0ff6119cfcd703cd156e0109ad5981',
   })
 
-  depends_on 'libunwind' # Better stack trace support
+  depends_on 'elfutils' # Needed for stack trace support
+
+  def self.patch
+    system "sed -i 's,/usr/bin/perl,#{CREW_PREFIX}/bin/perl,' strace-graph"
+  end
 
   def self.build
     system './configure',
-           '--with-libunwind',
+           '--with-libdw',
            "--prefix=#{CREW_PREFIX}",
            "--libdir=#{CREW_LIB_PREFIX}"
     system 'make'

--- a/packages/wl_clipboard.rb
+++ b/packages/wl_clipboard.rb
@@ -9,8 +9,16 @@ class Wl_clipboard < Package
   source_sha256 '3c7815986bb43c49912e1b64fb447bc31a80aa1326471dd575123ed6cd86e6be'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/wl_clipboard-c0109-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/wl_clipboard-c0109-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/wl_clipboard-c0109-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/wl_clipboard-c0109-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'd47f4caf0c77409dc90c3783b24b2033dd6b69388a492096b1048468f388e289',
+     armv7l: 'd47f4caf0c77409dc90c3783b24b2033dd6b69388a492096b1048468f388e289',
+       i686: '7f4ae7de3f2124c5439118e8fe463009eec9f096abd6e40ed3a364af30c1134e',
+     x86_64: '410bb77a3471d12c9babc4466486b7e2785f954fb2399bc5fbf51a555ce361db',
   })
 
   depends_on 'wayland_protocols' # xdg-shell support, depends on wayland

